setup.py

```diff
@@ -80,10 +80,5 @@ def ensure_scripts(linux_scripts):
     classifiers=PROJECT_CLASSIFIERS,
     scripts=PACKAGE_SCRIPTS,
     packages=PACKAGE_MODULES,
-    data_files=[
-        (
-            "Lib/site-packages/pymarkdown/resources",
-            ["pymarkdown/resources/entities.json"],
-        )
-    ],
+    include_package_data=True,
```

Comment:
Thanks for pointing these out.  Can you explain to me (any links would be helpful) the problem that this final change solves?

Reply:
Well, it is in the official documentation (sort of, [setuptools 57.4.0](https://setuptools.readthedocs.io/en/latest/userguide/datafiles.html))
Basically, `data_files` installs files relative to the used interpreter, while `include_data_files` takes files listed in the manifest and installs them relative to the package, which is more robust and the preferred way,